cloudinit/config/schemas/schema-cloud-config-v1.json

```diff
@@ -2478,8 +2478,8 @@
               "description": "The activation key to use. Must be used with ``org``. Should not be used with ``username`` or ``password``"
             },
             "org": {
-              "type": "integer",
-              "description": "The organization number to use. Must be used with ``activation-key``. Should not be used with ``username`` or ``password``"
+              "type": "string",
```

Comment:
As the value of this is ultimately is going to be passed as string to `subscription-manager` and in order to maintain backwards compatibility, I think we should maintain both types as valid and mark `integer` as deprecated.

Reply:
Thank you for the feedback. I will address this in a new commit!

Reply:
@pneigel-ca any updates on this?

Reply:
@holmanb No updates. If someone can handle in a new PR, I'm welcome to it. I haven't had the time to familiarize myself with the deprecation syntax and rebase with the changes to documentation.

Reply:
https://github.com/canonical/cloud-init/pull/5453

Reply:
@pneigel-ca Is 8532268+pneigel-ca@users.noreply.github.com your preferred email for the commit message?

Reply:
@holmanb Thanks for checking in - I didn't configure my email properly in git 😄 

`patrick.neigel@gmail.com` can be used.